python/school-timetabling/tests/test_feasible.py

```diff
@@ -0,0 +1,28 @@
+from timefold.solver import SolverFactory
+from timefold.solver.config import (SolverConfig, ScoreDirectorFactoryConfig,
+                                    TerminationConfig, Duration, TerminationCompositionStyle)
+
+from school_timetabling.domain import Timetable, Lesson
+from school_timetabling.constraints import school_timetabling_constraints
+from school_timetabling.demo_data import generate_demo_data, DemoData
+
+
+def test_feasible():
```

Comment:
Let's either create a new test or update this one to enable API calls, allowing us to test the API with supported Python versions.